### PR TITLE
fix parser

### PR DESCRIPTION
**Pull request by mattreex**
### Bug Fixes
Fix bug where whitespace is unintentionally left in name

### CODE REVIEW

File: tube/etl/indexers/base/parser.py [hunk 1]
```diff
@@ -105,7 +105,11 @@ def get_prop_type(self, fn, src, node_label=None, index=None):
 
     @staticmethod
     def get_src_name(props):
-        lst = [tuple(p.split(":")) if ":" in p else tuple([p, p]) for p in props]
+        # debug
+        lst = [
+            tuple(p.strip().split(":")) if ":" in p else tuple([p.strip()] * 2)
```
Comment (paulineribeyre) [inline tube/etl/indexers/base/parser.py:110]: i think this doesn't work if `p == "something :something"`

File: tube/etl/indexers/base/parser.py [hunk 2]
```diff
@@ -105,7 +105,10 @@ def get_prop_type(self, fn, src, node_label=None, index=None):
 
     @staticmethod
     def get_src_name(props):
-        lst = [tuple(p.split(":")) if ":" in p else tuple([p, p]) for p in props]
+        lst = [
+            tuple([p.strip() for p in p.split(":")]) if ":" in p else tuple([p.strip()] * 2)
```
Comment (paulineribeyre) [inline tube/etl/indexers/base/parser.py:109]:
```suggestion
            tuple([e.strip() for e in p.split(":")]) if ":" in p else tuple([p.strip()] * 2)
```

Comment (paulineribeyre) [reply tube/etl/indexers/base/parser.py:109]: for clarity.

**Review by paulineribeyre — COMMENTED** [review]
lgtm, but let's see if @thanh-nguyen-dang can review too

**Review by thanh-nguyen-dang — APPROVED** [review]
:+1:

**Pull request merged**